ui/src/pages/projects/add/index.tsx

```diff
@@ -0,0 +1,188 @@
+import { unwrapResult } from '@reduxjs/toolkit';
+import classNames from 'classnames/bind';
+import { Formik, FormikHelpers, FormikProps } from 'formik';
+import Link from 'next/link';
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import * as Yup from 'yup';
+
+import ImageUploadField from 'src/components/Form/ImageUploadField';
+import SelectField from 'src/components/Form/SelectField';
+import TextField from 'src/components/Form/TextField';
+import DashboardLayout from 'src/containers/DashboardLayout';
+import { RootState } from 'src/duck';
+import { fetchOrganization, OrganizationState } from 'src/duck/organization';
+import { createProject } from 'src/duck/project';
+
+import style from './style.module.scss';
+
+const c = classNames.bind(style);
+
+const Header: React.FC = () => (
+  <div className={c('header')}>
+    <span>
+      <Link href="/projects">
+        <a>Projects > </a>
+      </Link>
+      Create
+    </span>
+  </div>
+);
+
+const AddProject: React.FC<FormikProps<FormValues>> = (p) => {
+  const dispatch = useDispatch();
+  const { organizations, isFetching: isFetchingOrganizations } = useSelector<
+    RootState,
+    OrganizationState
+  >((state) => state.organization);
+
+  useEffect(() => {
+    dispatch(fetchOrganization());
+  }, []);
```

Comment:
1. Admin should see full list of organizations in this form
2. User should see only current organization